src/db/products-controllers.ts

```diff
@@ -0,0 +1,43 @@
+import {Product} from './products';
+
+export const getProducts = async (ctx) => {
+    ctx.ok(await Product.find());
```

Comment:
should not be in db folder! create controllers folder inside http folder.
moreover, create here (inside db folder) file that containes all your db queries. (acctually it should be inside dall folder but it isnt important)